guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -39,14 +39,20 @@ On the Discovered Hosts page, power off and then delete the discovered hosts.
 From the *Select an Organization* menu, select each organization in turn and repeat the process to power off and delete the discovered hosts.
 Make a note to reboot these hosts when the upgrade is complete.
 ifdef::satellite[]
-. Ensure that the {Project} Maintenance repository is enabled:
+. Ensure that the {Project} Maintenance repository and the {Project} {ProjectVersion} repository are enabled:
 +
 [options="nowrap" subs="attributes"]
 ----
-# subscription-manager repos --enable \
-{RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+# subscription-manager repos \
+--enable {RepoRHEL8ServerSatelliteMaintenanceProductVersion} \
+--enable {RepoRHEL8ServerSatelliteServerProductVersion}
 ----
 
+[NOTE]
+====
+If you find further issues on upgrading the {ProjectServer}, please refer {MultiBaseURL}release_notes/index#assembly_introducing-red-hat-satellite_sat6-release-notes#ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues] in the Release Notes as a workaround.
```

Comment:
```suggestion
If you find further issues on upgrading the {ProjectServer}, please refer {MultiBaseURL}release_notes/assembly_introducing-red-hat-satellite_sat6-release-notes#ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues] in the Release Notes as a workaround.
```

Reply:
Ahh yes! I was wondering why it's not getting redirected to Known Issues. This fixed it :heart: 
Thanks @Lennonka